day-one/create-resource-from-service.mdx

```diff
@@ -1,6 +1,200 @@
 ---
 title: 'Create Resource from Service'
 sidebarTitle: 'Create from a Service'
-description: 'Guidance for enterprise developers on creating services using Amplication, including .NET and Node.js options.'
+description: "Quickly generate a production-grade service using our opinionated, pre-configured Node.js or .NET code generation engines."
 icon: 'code-simple'
----
\ No newline at end of file
+---
+
+<Frame>
+  <img src="/images/day-one/service.png" />
+</Frame>
+
+You may already have established technology standards, but Amplication provides an additional resource option called a **Service**.
+Services generate a complete, opinionated codebase in Node.js or .NET that you can customize extensively to fit into your existing software ecosystem.
```

Comment:
If you want to build services with Node.js or .NET and haven't yet developed your own standards and opinions, you can adopt Amplication's opinionated code generation for Node.js or .NET.

Reply:
Is this how we want to frame it, @yuval-hazaz? Blueprints **OR** Services? If I'm understanding correctly, folks can mix and match Blueprint Resources and Service resources?

Reply:
yes ofcourse - you can mix.
also - you can create services from blueprint
so the term "service" in amplication referring to the opinionated services but also with blueprint you can create services